health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/BannerCard.js

```
@@ -0,0 +1,136 @@
+  const Cell = ({ name, value, symbol }) => {
+    let formattedValue = value;
+    if (symbol === "number") {
+      formattedValue = Digit.Utils.dss.formatter(formattedValue, "number", value?.denomination, true, t);
+    }
+    if (symbol === "percentage") {
+      formattedValue = Digit.Utils.dss.formatter(formattedValue, "percentage", value?.denomination, true, t).replace(" ", "");
+    }
+
+    return (
+      <div className="digit-banner-cell">
+        <div className="digit-banner-cell-sub-text">{t("DSS_" + name?.replaceAll(" ", "_").toUpperCase())}</div>
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)